sample/src/main/assets/json.form/single_form.json

```diff
@@ -1198,6 +1198,17 @@
         "text_color": "#FFC100",
         "toaster_type": "warning"
       },
+      {
+        "key": "toaster_notes",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "person_attribute",
+        "openmrs_entity_id": "toaster_notes",
+        "type": "toaster_notes",
+        "icon_visible": false,
+        "background_color": "#ebebeb",
+        "text": "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+        "text_color": "#000000"
```

Comment:
Black should be the default color. If that's not the case on the factory please update.  this will mean you don't need the `text_color` key when using the default color.